ably-ios/ARTDataQuery+Private.h

```diff
@@ -16,4 +17,10 @@ ART_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ARTRealtimeHistoryQuery ()
+
+@property (strong, readwrite) ARTRealtimeChannel *realtimeChannel;
```

Comment:
Right, but as soon as the query is gone (ie. as soon as the call site returns) the channel will lose its only reference left and all will be good. How can that leak?